src/firefly/js/drawingLayers/Catalog.js

```diff
@@ -162,16 +191,16 @@ function computeDrawLayer(tableData, columns) {
     return ptAry;
```

Comment:
Very minor: you assign a value to variable just to return. If you want to document the code this way, it would be better to use JSDoc.